### PR TITLE
streamline whole documentation and added advanced_configuration

### PR DESCRIPTION
**Pull request by OskarStark**
sonata-project/SonataAdminBundle#3310

### CODE REVIEW

File: Resources/doc/reference/installation.rst [hunk 1]
```diff
@@ -27,69 +25,72 @@ Installation
...
+==================      ============================================================================
+**locales**             The list of your frontend locales in which your models will be translatable.
+**default_locale**      The locale, loaded in your forms by default.
+==================      ============================================================================
 
-Here is an example with phpcr :
+.. note::
 
-.. code-block:: yaml
+    If you are using ``SonatAdmin`` with SonataDoctrineORMAdminBundle_ you should read the :doc:`ORM Chapter <reference/orm>`,
```
Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: is
```rst
:doc:`ORM Chapter <reference/orm>`
```

corect?

Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: or
```rst
:doc:`ORM Chapter <orm>`
```

because the file is in the same folder?

Comment (javiereguiluz) [inline Resources/doc/reference/installation.rst]: I don't know ... but in Symfony docs we always use this syntax (note the leading `/`):

```
:doc:`ORM Chapter </reference/orm>`
```

Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: thank you

**Pull request merged**